src/sql/base/browser/ui/infoBox/media/infoBox.css

```diff
@@ -38,7 +37,17 @@
 }
 
 .infobox-text {
-	flex: 1 1 auto;
-	padding: 10px 0;
+	flex: 0 0 auto;
+	padding: 10px 0px 0px 0px;
 	user-select: text;
+	overflow: scroll;
```

Comment:
Do we want these changes for all infoboxes, even non-clickable ones? (mostly the max-width. The overflow change seems fine)